.github/workflows/test.yml

```diff
@@ -33,6 +38,20 @@ jobs:
       - name: Install dependencies
         if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
         run: make setup
+
+      # Build artifacts
+      - name: Event name
+        run: echo "github.event_name=${{ github.event_name }}"
+      - name: Format all the cairo files
+        run: make format
+      - name: Commit changes
+        # Don't run on pull request events. Commit only when there are changes, otherwise it continues
+        if: github.event_name != 'pull_request'
```

Comment:
I think this doesn't help the reader know the other events.

If there are like 5-6 `github.event_name`, then I guess this is fine you wouldn't write `if: github.event_name == 'push' || ....` -> then I guess we can probably list them in comment

If there's only 1 or 2 we're targeting, could we prefer the `==` notation?

Reply:
That's true we only have `push` and `pull_request` event triggers we're targeting and it will be more explicit to say we check and commit the formatting every `push` event 👍  I've updated this to `== 'push'` notation, thanks @Eikix